webpack.mix.js

```diff
@@ -4,5 +4,7 @@ const plugins = [require("tailwindcss")];
 
 mix
     .postCss("src/css/tailwind.css", "assets/css", plugins)
-    .copy('src/js/', 'assets/js/')
-    .copy('src/css/munim.css', 'assets/css');
+    .copy('src/js/dashboard.js', 'assets/js/dashboard.js').minify('assets/js/dashboard.js')
+    .copy('src/js/script.js', 'assets/js/script.js').minify('assets/js/script.js')
+    .copy('src/css/munim.css', 'assets/css')
+    .copy('node_modules/clipboard/dist/clipboard.min.js', 'assets/js');
```

Comment:
we can improve this 

1. copy un-minifed file for clipboard 
2. copy whole folder instead of individual files `mix.copyDirectory('src/js', 'assets/js');`
3. minify all files in folder instead of individual files `mix.minify('assets/js');`

Reply:
I searched a lot and found that that `mix.minify` only take one file name at once.

Reply:
for now, let's use the array of file names, later we can auto-generate file list from folder.

```
const js_assets = [
	'src/js/dashboard.js',
	'src/js/script.js'
];

mix.minify(js_assets);
```

Reply:
Okay